### PR TITLE
MTA-547: Document: Presenting data in more readable format

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-547

MTA 6.2.0

Preview: https://deploy-preview-718--windup-documentation.netlify.app/docs/web-console-guide/master/#viewing-analysis-details_user-interface-guide

### CODE REVIEW

**Review by dymurray — APPROVED** [review]

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc [hunk 1]
```diff
@@ -106,5 +106,14 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time.  With more cluster capacity, more analysis processes can be executed in parallel.
 ====
-
 . When analysis is complete, you can click the *Report* link to see the results of the analysis.
+
+[id="viewing-analysis-details_{context}"]
+== Viewing analysis details
+
+You can view the details of an analysis by clicking the options menu ({kebab}) and selecting *Analysis details*. The details are displayed in the *Analysis details for customers* window. You can choose either YAML or JSON format.
```
Comment (kpunwatk) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: Hi @sbeskin-redhat I feel we are conveying the same thing twice(Repetitive), can't we somehow re-frame both this line in single sentence. `You can view the details of an analysis by clicking the options menu ({kebab}) and selecting *Analysis details*. The details are displayed in the *Analysis details for customers* window. `

Format can be plural

Comment (sbeskin-redhat) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: @kpunwatk 
I am not sure I understand what you mean. What is repetitive here?

Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: @sbeskin-redhat What i meant was this line `The details are displayed in the *Analysis details for customers* window` not sure about this term `Analysis details for customers` if it's something different then it's okay to mention in different sentences. If this means the same window which opens after clicking the Analysis details then i think we could add in same line as it feels `view the details` and `details are displayed` means the same start for the line.

Comment (sbeskin-redhat) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: @kpunwatk 
In the first sentence, we tell the user which option they should use to view the details - the 'Analysis details' option from the menu.
In the second sentence, we tell the user what happens when they click that option - the 'Analysis details for customers' window opens, and it displays the details.

Comment (kpunwatk) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: Ah, Okay I see. Thanks for the clarification @sbeskin-redhat 
 I will approve the PR then. Thanks!

**Review by kpunwatk — APPROVED** [review]
LGTM :+1:

**Comment by RichardHoch** [comment]
LGTM

File: docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc (hunk 1 above)
Comment (anarnold97) [inline docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc:114]: @sbeskin-redhat  If you look at the rest of the MTA documentation, you will notice that the format for options menu kebab is **Options menu {kebab}**. Please make this change. And in future, please ensure you are consistent. If you are unsure about format used, check the existing documentation.

Comment (anarnold97) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: @sbeskin-redhat - 
image
image

Comment (sbeskin-redhat) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: This menu does not have any visible name in the UI.

Comment (anarnold97) [reply docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc]: @sbeskin-redhat  if there is no title when you hover over the menu kebab, that is a UI issue, and you should report it. There might not be any text to be picked up by a Screen Reader, which is an accessibility issue.

**Pull request merged**